projects/storefrontlib/shared/components/table/table.service.ts

```diff
@@ -79,18 +79,19 @@ export class TableService {
     type: string,
     breakpoint: BREAKPOINT,
     defaultStructure?: ResponsiveTableConfiguration
-  ): TableStructureConfiguration {
+  ): TableStructureConfiguration | undefined {
     if (!this.config.table?.[type]) {
-      return null;
+      return undefined;
```

Comment:
Maybe null plays a big role for the table service

I believe we can keep it to `return null` and typed `as | null` assuming that it wasn't like

```
    if (!this.config.table?.[type]) {
      return;
```